data_steward/spec/pages/index.md

```diff
@@ -12,6 +12,18 @@ expected to send data sets to the program).
 This document is intended primarily for personnel (who we refer to as _data stewards_) who hold the responsibility 
 of submitting EHR data on behalf of HPOs.
 
-# Timeline
+## Timeline
 
 _Sites will be expected to upload datasets on a quarterly basis. Associated deadlines will be posted here._ 
+
+## EHR Uploading
+ - Sites are allowed to upload EHR data after it's ISA has been executed.
+ - Maintain accurate and updated site contact information with the DRC
+ - Other items here that are important.......
+
+## Data Sprint Goals
+__For new sites to All Of Us the first step in submitting EHR data is to participate in a Data Sprint.__
+1. Become Familiar with the OHDSI Vocabulary. Based on the previous data sprint, an abridged version of the OMOP vocabulary was created. This data sprint will assess sites capabilities in converting their coded data into OMOP concept IDs.
```

Comment:
Become familiar with the OHDSI Vocabulary. Based on the previous data sprint, an abridged version of the OMOP vocabulary was created. This data sprint will assess sites capabilities in converting coded data into OMOP concept IDs.